_docs/Components-Box.md

```diff
@@ -6,7 +6,7 @@ title: "Components: Box"
 ## Abstract
 
 The box component is a core component of Qabel. It serves as a file storage for multiple clients and allows clients to share access to stored files.
-The files are stored on AWS S3 and encrypted.
+The files are encrypted and stored by the Qabel Block server on AWS S3.
```

Comment:
The encrypted files are stored by the QBS, right?
They are not encrypted by the QBS. The wording might be confusing.